recipes/seqan3/meta.yaml

```
@@ -0,0 +1,32 @@
+{% set name = "SeqAn3" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/seqan/{{ name|lower }}/releases/download/{{ version }}/seqan-3.0.0-with-submodules.tar.gz
+  sha256: ba8f0fd5d19289f95823235b69fdc5a12aba3578292463eecf750722e8662bd3
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+
+test:
+  commands:
+    - test -e ${PREFIX}/include/seqan/align.h
+    - test -e ${PREFIX}/share/cmake/seqan/seqan-config.cmake
```

This path changed. And I'm not to sure if the `seqan3-config.cmake` should be in the `/share/cmake/seqan/` folder.

```suggestion
    - test -e ${PREFIX}/share/cmake/seqan3/seqan3-config.cmake
```